.github/workflows/test.yaml

```diff
@@ -0,0 +1,102 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+    test:
+        name: PHP ${{ matrix.php-version }} + Symfony ${{ matrix.symfony-version }}
+
+        runs-on: ubuntu-18.04
+
+        continue-on-error: ${{ matrix.experimental }}
+
+        strategy:
+            matrix:
+                include:
+                    - php-version: '7.1'
+                      symfony-version: '^4.3'
+                      composer-version: v1
+                      stability: stable
+                      coverage: none
```

Comment:
why did you disabled the coverage ?

Reply:
The coverage is enabled only for the single build with PHP 7.4 and Symfony 5.0 (see line https://github.com/eMAGTechLabs/RabbitMqBundle/pull/45/files#diff-245392b692a50c38ecab4381b118862db514035c10983f3bd4f4b7f1f4be4692R44 and report https://github.com/javer/RabbitMqBundle/runs/2081131219?check_suite_focus=true#step:12:30), because:
* I see no point to send the same coverage 6 times in a row for the same commit to Scrutinizer
* I have no idea how Scrutinizer can distinguish these 6 coverages, because they all are for the same commit
* PHPUnit 8 does not support code coverage on PHP 8, see https://github.com/javer/RabbitMqBundle/runs/2070283209?check_suite_focus=true#step:10:15